wrappers/python/pyrs_device.cpp

```diff
@@ -60,7 +60,7 @@ void init_device(py::module &m) {
 
             if (!depth_sens.supports(RS2_CAMERA_INFO_PHYSICAL_PORT))
             {
-                throw librealsense::invalid_value_exception("Device does not support checking metadata with thi API");
+                throw librealsense::invalid_value_exception("Device does not support checking metadata with this API");
```

Comment:
I don't think we should use `librealsense` errors from Pybind... we should use pybind errors:
https://pybind11.readthedocs.io/en/stable/advanced/exceptions.html
I'd suggest a plain old `std::runtime_error`.